### ISSUE TITLE
Edit of an executable is for some usecases hard to follow

### ISSUE THREAD

**Issue by vmcj** [DOMjudge/domjudge#1783]
If we navigate to https://domjudge/jury/executables and wish to edit the runscripts we first need to click on a link which is not clearly a link. Problem here is we have 2 different edit screens where I think 1 would be good enough.

@eldering @meisterT what is the legacy here?

**Comment by meisterT** [comment]
I agree it is annoying and started simplifying things a while back and kept https://github.com/DOMjudge/domjudge/issues/1526#issuecomment-1107525869 open for this exact reason.
